fontique/Cargo.toml

```diff
@@ -46,10 +46,9 @@ windows = { version = "0.58.0", features = ["implement", "Win32_Graphics_DirectW
 windows-core = { version = "0.58", optional = true }
 
 [target.'cfg(target_vendor = "apple")'.dependencies]
-core-text = { version = "20.1.0", optional = true }
-core-foundation = { version = "0.9.4", optional = true }
-objc2 = { version = "0.5.2", optional = true }
-objc2-foundation = { version = "0.2.2", features = ["NSArray", "NSEnumerator", "NSPathUtilities", "NSString"], optional = true }
+objc2-foundation = { version = "0.3.0", optional = true, default-features = false, features = ["alloc", "NSArray", "NSEnumerator", "NSPathUtilities", "NSString"] }
+objc2-core-foundation = { version = "0.3.0", optional = true, default-features = false, features = ["CFBase"]  }
+objc2-core-text = { version = "0.3.0", optional = true }
```

Comment:
I still need to turn off `default-features` here and then update and re-format the above.